main/parse.c

```diff
@@ -1158,6 +1189,28 @@ static void lazyInitialize (langType language)
 /*
 *   Option parsing
 */
+#define COUNT(D) (sizeof(D)/sizeof(D[0]))
+
+static void lang_def_flag_file_kind_long (const char* const optflag, const char* const param, void* data)
```

Comment:
Loog at lregex.c, I use `_` for flag related callback functions.
So the name lang_def_flag_file_kind_long is natural for me.

How about following rules?
About file static varaibles and functions we can choose any style.
About extern functions we should use camel style.

> $  grep _  *.h | grep -v define | grep -v undef | grep -v TAG| grep -v ifdef | grep -v off_t | grep -v readtags | grep -v endif| grep -v size_t | grep -v ifndef | grep -v '[A-Z]_[A-Z]' | wc -l
 grep _  *.h | grep -v define | grep -v undef | grep -v TAG| grep -v ifdef | grep -v off_t | grep -v readtags | grep -v endif| grep -v size_t | grep -v ifndef | grep -v '[A-Z]_[A-Z]' | wc -l
44

Most of all 44 hits are introduced by me. They can be fixed.

For field name, I would like to use `_`.

Do the same against *.c in parsers directlry, I got 2368 hits.